### PR TITLE
Wastabir's new work

### PR DESCRIPTION
**Pull request by dividedbyzero360**
I have added a new cool functionality

### CODE REVIEW

File: new_file.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-hello git
\ No newline at end of file
+hello world
+
+hello sadid
```
Comment (Shadid-dev) [inline new_file.txt]: capitalze the letter S

**Review by Shadid-dev — CHANGES_REQUESTED** [review]
Please see my comments

File: new_file.txt (hunk 1 above)
Comment (dividedbyzero360) [reply new_file.txt]: You are right, let me make the fix.

Comment (dividedbyzero360) [reply new_file.txt]: @Shadid-dev  I have made the change and the commit is https://github.com/dividedbyzero360/git_demo/pull/1/commits/4d991d3e5da85d2d1e01e7856c37c5991becb921

**Review by Shadid-dev — APPROVED** [review]
Lgtm

**Pull request merged**